code/modules/jobs/job_types/legion.dm

```diff
@@ -242,7 +242,7 @@ Access
 	ADD_TRAIT(H, TRAIT_HARD_YARDS, src)
 	ADD_TRAIT(H, TRAIT_LIFEGIVER, src)
 	ADD_TRAIT(H, TRAIT_FAST_PUMP, src)
-
+	box_two = /obj/item/storage/survivalkit/medical/legion
```

Comment:
What's this random box_two up here?

Reply:
Leftover that slipped through. Corrected.